examples/stackdriver-trace/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "example-stackdriver-trace",
   "private": true,
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Comment:
What will it happens since the exporter moved (https://github.com/open-telemetry/opentelemetry-js/pull/859), should example move as well ?

Reply:
It is already removed, the branch is just out of date.